cdap-ui/app/cdap/components/HttpExecutor/utilities.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { List, Map } from 'immutable';
+
+import { IRequestHistory } from 'components/HttpExecutor/RequestHistoryTab';
+import moment from 'moment';
+
+export function getDateID(date: Date) {
+  return moment(date).format('dddd, MMMM d, YYYY');
```

Comment:
i think this is supposed to be `MMMM D`. `d` is day of the week, not month

Reply:
Done